qmri_tb1tfl/sub-01/fmap/sub-01_acq-anat_TB1TFL.json

```diff
@@ -0,0 +1,12 @@
+{
+  "MagneticFieldStrength": "3",
+  "Manufacturer": "Siemens",
+  "ManufacturerModelName": "TrioTim",
+  "PulseSequenceType": "GR",
+  "SequenceName": "tfl_b1_map",
+  "RepetitionTime": "6.8",
```

Comment:
I think:

```suggestion
  "RepetitionTimeExcitation": "6.8",
```

Reply:
Can't we use RT and RTE interchangeably for anat if RTP is not present?

Reply:
Bare RT is only defined for functional, and I think it's good to keep it that way due to the ambiguity of RTE and RTP. While we might want to allow functional to have synonyms in order to bring it into line with the rest of MRI while maintaining backwards compatibility, it's not a good idea to add synonyms to places they didn't previously apply. This would just make work for tool developers.